astropy/stats/funcs.py

```diff
@@ -1325,9 +1325,9 @@ def kuiper_false_positive_probability(D, N):
     if D < 0.0 or D > 2.0:
         raise ValueError("Must have 0<=D<=2 by definition of the Kuiper test")
 
-    if D < 2.0 / N:
+    if D < 2.0 / N:  # noqa: SIM300
```

Comment:
If `SIM300` works so badly, I think we should not use it. It is really distracting to have `noqa` in the code, so it better be meaningful.